bitnami/redis/templates/scripts-configmap.yaml

```diff
@@ -140,6 +145,14 @@ data:
     . /opt/bitnami/scripts/libvalidations.sh
     . /opt/bitnami/scripts/libfile.sh
 
+    myip=$(hostname -i)
+    
+    #If two IPs, use 2nd one (IPv4 in dualstack environment)
+    if [[ "$myip" =~ ( ) ]]
```

Comment:
It is a bit hard to understand this specific line, I would suggest to rewrite it like:

```bash
if [[ "$myip" = *" "* ]]; then
```

Reply:
done